server/models/User.js

```diff
@@ -93,7 +93,7 @@ class User extends Model {
             .subject('You requested for a password reset.')
             .data({
                 name: this.firstName,
-                url: `${config.server.url}/api/v1/auth/reset/${token}`
+                url: `${config.server.url}/reset?token=${token}`
```

Comment:
Please let's make this url as such : `/reset-password/${token}`

On the frontend, get the `match` object from props in the component and you'll get direct access to the component. No need for `new URLSearchParams` anymore. Also, on the frontend you need to register the route accordingly. `<Route path="/reset-password/:token" component={ResetPassword} />`

Reply:
I already tried using the recommendation you gave before now but it wasn't loading the component. I will appreciate a way out of that